.github/workflows/build.yml

```
@@ -131,17 +192,54 @@ jobs:
           # Workaround for protoc https://github.com/microsoft/onnxruntime/issues/16238#issuecomment-1590398821
           brew install protobuf@21
 
-      # ONNX Runtime v1.15.1 requires CMake 3.26 or higher.
+      # ONNX Runtime v1.16.1 requires CMake 3.27 or higher.
       - name: Install CMake
         if: steps.cache-build-result.outputs.cache-hit != 'true' && startsWith(matrix.os, 'ubuntu')
         env:
-          CMAKE_VERSION: 3.26.4
+          CMAKE_VERSION: 3.27.7
         run: |
           wget -O cmake.sh "https://github.com/Kitware/CMake/releases/download/v${{ env.CMAKE_VERSION }}/cmake-${{ env.CMAKE_VERSION }}-linux-x86_64.sh"
           sudo bash cmake.sh --skip-license --prefix=/usr/local
 
+      - name: Set up CUDA
+        if: steps.cache-build-result.outputs.cache-hit != 'true' && matrix.cuda_version && matrix.cudnn_url
+        uses: Jimver/cuda-toolkit@v0.2.11
+        with:
+          cuda: ${{ matrix.cuda_version }}
+          sub-packages: ${{ matrix.cuda_sub_packages }}
+          method: network
+
+      - name: Set `$CUDA_HOME`
+        if: steps.cache-build-result.outputs.cache-hit != 'true' && matrix.cuda_version && matrix.cudnn_url
+        run: echo "CUDA_HOME=$CUDA_PATH" >> "$GITHUB_ENV"
+
+      - name: Set up cuDNN
+        if: steps.cache-build-result.outputs.cache-hit != 'true' && matrix.cuda_version && matrix.cudnn_url
+        run: |
+          mkdir download
+
+          if [ ${{ runner.os }} = Windows ]; then
+            curl -L "${{ matrix.cudnn_url }}" > download/cudnn.zip
+            unzip download/cudnn.zip -d download
+            rm download/cudnn.zip
+          else
+            curl -L "${{ matrix.cudnn_url }}" > download/cudnn.tar.gz
+            tar xvf download/cudnn.tar.gz -C download/
+            rm download/cudnn.tar.gz
+          fi
+
+          cudnn_path=$(find download -maxdepth 1 -name 'cudnn-*')
+          : "${cudnn_path:=./download/cuda}" # https://developer.download.nvidia.com/compute/redist/cudnn/v8.2.4/cudnn-11.4-linux-x64-v8.2.4.15.tgz
+          if [ ${{ runner.os }} = Windows ]; then
+            cudnn_path=$(cygpath -wa "$cudnn_path")
+          else
+            cudnn_path=$(realpath "$cudnn_path")
+          fi
+
+          echo "CUDNN_HOME=$cudnn_path" >> "$GITHUB_ENV"
+
       - name: Configure build environment

```

x64以外の、という意図を足しておくとミスが減るかも？
```suggestion
      - name: Configure build environment for not x64

```

---

"for non-x86_64 Linux"としました。
[`3de493b` (#26)](https://github.com/VOICEVOX/onnxruntime-builder/pull/26/commits/3de493b3e089e6947b0f0c81aa007ecfe6771146)